test/it/migrations/changelogs/notification/MakeParsedDetailsOptionalISpec.scala

```diff
@@ -0,0 +1,116 @@
+package migrations.changelogs.notification
+
+import base.IntegrationSpec
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.mongodb.{MongoClient, MongoClientURI}
+import com.mongodb.client.{MongoCollection, MongoDatabase}
+import com.mongodb.client.model.Indexes
+import migrations.changelogs.notification.MakeParsedDetailsOptionalSpec._
+import org.bson.Document
+import org.mongodb.scala.model.IndexOptions
+import org.scalatest.BeforeAndAfterEach
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.Application
+
+class MakeParsedDetailsOptionalSpec extends IntegrationSpec with GuiceOneServerPerSuite with BeforeAndAfterEach {
```

Comment:
Nit: I think there is a typo in the file name.

Reply:
:+1: